ricecooker/classes/files.py

```diff
@@ -417,7 +417,7 @@ def process_file(self):
             self.filename = self.download_subtitle()
             config.LOGGER.info("\t--- Downloaded subtitle {}".format(self.filename))
             return self.filename
-        except FileNotFoundError:
+        except (FileNotFoundError, youtube_dl.utils.DownloadError):
```

Comment:
I need to make a note that the reason I added this error was because there was some youtube ids that `youtube_dl` could not find. They have been removed from youtube. @ivanistheone